app/views/content/life-as-a-teacher/teaching-as-a-career/what-its-like-to-be-a-teacher/_article.html.erb

```diff
@@ -40,6 +40,12 @@
  <p> When you're training, you'll work with a teacher who will help you plan and may share some of their own planning with you. Over time, with the help of other colleagues, you'll get a better feel for what makes a good lesson and build a bank of resources you can reuse to cut down on your workload.</p>
    <p>Teachers are entitled to 10% of their timetable to do planning, preparation, and assessment (PPA). Depending on the school, this could mean that you have a couple of free periods next to each other, or they could be spread across the week.</p>
  <%= render 'content/shared/quotes/quote_food_helen' %>
+
+ <h3>Adjustments and accessibility</h3>
+ <p>Every day teaching can look different, depending on the needs of the school, pupils, and the teacher.</p>
```

Comment:
Not sure I quite understand the first half of this sentence `Every day teaching can look different` ?